soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -484,121 +484,121 @@ void InitTrickNames() {
   trickNameTable[0xC1] = {
      Text{"Ballad of the Goddess", "Chant de la déesse", "Cántico de la Diosa"},
      Text{"Song of Healing", "Chant de l'apaisement", "Canción de curación"},
-     Text{"Bolero of Fire", "Boléro du feu", "Bolero del fuego"}};
+     Text{"Song of the Hero", "Chant du héros", "Canción del héroe"}};
```

Comment:
Old was forcing the player to pay attention on the GI item and not the title

Reply:
Yes, and fixing that is a major motivation of these changes, as these names are used for scrubs and merchant hints too, which do not have graphics.